wdio.conf.js

```diff
@@ -129,7 +130,11 @@ exports.config = {
     // The only one supported by default is 'dot'
     // see also: https://webdriver.io/docs/dot-reporter.html
     reporters: ['spec'],
-    //
+    services: [
+        ['chromedriver', {
```

Comment:
this should go into the build config file where the chromedriver service is already defined: https://github.com/webdriverio/cucumber-boilerplate/blob/master/wdio.BUILD.conf.js#L23-L26 .. I would recommend to remove this change.